### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by mgucma**

### CODE REVIEW

File: src/main/java/com/marek/onlinebookstore/service/impl/BookServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import com.marek.onlinebookstore.entity.Book;
+import com.marek.onlinebookstore.repository.BookRepository;
+import com.marek.onlinebookstore.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+
```
Comment (beatarjn) [inline src/main/java/com/marek/onlinebookstore/service/impl/BookServiceImpl.java]: Pls remove empty line

File: src/main/java/com/marek/onlinebookstore/entity/Book.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+
+@Entity
+public class Book {
+
```
Comment (beatarjn) [inline src/main/java/com/marek/onlinebookstore/entity/Book.java:12]: Pls remove empty line

File: src/test/java/com/marek/onlinebookstore/BookRepositoryTests.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class BookRepositoryTests {
+
```
Comment (beatarjn) [inline src/test/java/com/marek/onlinebookstore/BookRepositoryTests.java]: Pls remove empty line

File: src/test/java/com/marek/onlinebookstore/BookRepositoryTests.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    @Autowired
+    private BookRepository bookRepository;
+
+    @Test
+    void testCreateReadUpdateDelete() {
```
Comment (beatarjn) [inline src/test/java/com/marek/onlinebookstore/BookRepositoryTests.java]: Unit tests should be created separately for each method.

**Review by beatarjn — CHANGES_REQUESTED** [review]

File: src/main/java/com/marek/onlinebookstore/service/impl/BookServiceImpl.java (hunk 1 above)
Comment (mgsorigo) [reply src/main/java/com/marek/onlinebookstore/service/impl/BookServiceImpl.java:12]: oki

**Review by MateuszRostek — APPROVED** [review]

**Review by MateuszRostek — CHANGES_REQUESTED** [review]
Remove all tests.

**Review by MateuszRostek — APPROVED** [review]
Good job

**Pull request merged**